lib/grandpa/grandpa.go

```diff
@@ -258,13 +260,41 @@ func (s *Service) updateAuthorities() error {
 	// setting to 0 before incrementing indicates
 	// the setID has been increased
 	s.state.round = 0
+
+	s.sendTelemetryAuthoritySet()
+
 	return nil
 }
 
 func (s *Service) publicKeyBytes() ed25519.PublicKeyBytes {
 	return s.keypair.Public().(*ed25519.PublicKey).AsBytes()
 }
 
+func (s *Service) sendTelemetryAuthoritySet() {
+	authorityID := s.keypair.Public().Hex()
+	authorities := make([]string, len(s.state.voters))
+	for i, voter := range s.state.voters {
+		authorities[i] = fmt.Sprintf("%d", voter.ID)
+	}
+
+	authoritiesBytes, err := json.Marshal(authorities)
+	if err != nil {
+		logger.Debugf("could not marshal authorities: %s", err)
```

Comment:
Shouldn't this be a warning or error at least? Not being able to marshal the authorities can be quite a problem right?